docs/gettingStarted/plugins/gnuradio/Windows/condaRecipe.rst

```diff
@@ -0,0 +1,34 @@
+Build using conda recipe
+========================
+
+.. important::
+
+   Compared to standard build procedure, building plugin package using conda-build tool takes more time (5-8 min.) and resources. For development purposes it is advised to build from source using cmake. See :ref:`windows-plugin-build-ref`.
+
+.. hint::
+   
+   Check out radioconda and conda environment setup. See :ref:`radioconda-setup-ref`.
+
+Gnuradio-limesuiteng plugin for GNURadio 3.10.9.2 (Python 3.12) can also be built using conda recipe. Plugin recipe files are provided in ``<project>\plugins\gr-limesuiteng\.conda\`` directory.
```

Comment:
no need to specify exact versions everywhere in general descriptions. Versions info is provided within the build scripts.